addons/launcher-addon/src/main/java/io/fabric8/launcher/addon/catalog/RhoarBooster.java

```diff
@@ -0,0 +1,75 @@
+package io.fabric8.launcher.addon.catalog;
+
+import java.util.Map;
+
+import io.openshift.booster.catalog.Booster;
+import io.openshift.booster.catalog.BoosterFetcher;
+
+public class RhoarBooster extends Booster {
```

Comment:
I´d rather have everything in the `catalog` package as a `rhoar` sub package in launcher-booster-catalog-service